src/bot.py

```diff
@@ -229,6 +229,7 @@ async def on_message(message):
     temp=[]
     ctx = await bot.get_context(message)
     print(message.content)
+    print(message.author.id)

```

Comment:
Are the print statements necessary in the main branch or are they just for debugging. They need to be removed if they are just for debugging.

Reply:
They are just for debugging